nodehid.js

```diff
@@ -59,17 +67,18 @@ HID.prototype.close = function close() {
     this._raw.close();
     this._closed = true;
 };
+
 //Pauses the reader, which stops "data" events from being emitted
 HID.prototype.pause = function pause() {
     this._paused = true;
 };
 
 HID.prototype.read = function read(callback) {
-    if (this._closed) {
-    throw new Error('Unable to read from a closed HID device');
-  } else {
-    return this._raw.read(callback);
-  }
+	if (this._closed) {
+		throw new Error('Unable to read from a closed HID device');
+	} else {
+		return this._raw.read(callback);
+	}
```

Comment:
It looks like I may have a tabs vs. spaces issue. Is 4-space indent the code standard?